data-raw/standard-format-data-prep/rst_trap_standard_format.Rmd

```diff
@@ -236,21 +235,21 @@ butte_trap_clean_pre_2015 <- butte_rst %>%
   select(-gear_id)
 
 
```

Comment:
When the NA_real_ were changed to NA this no longer worked for me. Is it working for you?

Reply:
Updated all the NAs within case_when() statements to use as.Date(NA) or as_hms(NA) or as.character(NA) or as.logical(NA) depending on data types.